acceptance_tests/features/DSP-18.feature

```diff
@@ -0,0 +1,5 @@
+# Feature: DSP-18

```

Comment:
Please only check in completed features. Generally it's a bad idea to commit uncompleted features so as a rule we don't commit commented out code.